test/io/bharshbarg/overflow.good

```diff
@@ -0,0 +1 @@
+IOError: Input/output error (line longer than maxSize in channel.readLine(arg : [] uint(8)) with path "unknown" offset 0)
```

Comment:
I don't think Input/output error is the right error code here. It's really used for hardware failures. Maybe BadFormatError would be better?

Reply:
How does this look?
```
BadFormatError: bad format (line longer than maxSize in channel.readLine(arg : [] uint(8)) with path "unknown" offset 8)
```

Reply:
Remember to fix all 3 readLine functions for it. Would be reasonable to ask on the readLine issue also in case others want to weigh in on which error is thrown here. But I think this PR can proceed.